recipes/neuralforecast/meta.yaml

```diff
@@ -0,0 +1,76 @@
+{% set name = "neuralforecast" %}
+{% set version = "0.0.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/neuralforecast-{{ version }}.tar.gz
+  sha256: 31e27969195f03b9ccfb6cab39b339c8c1d52d4048ce727d3a6ffab1dceb2e94
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - python >=3.7
+    - fastcore
+    - gdown
+    - hyperopt
+    - matplotlib-base
+    - numba
+    - openpyxl
+    - pandas
+    - pip
+    - py7zr >=0.18.1
+    - pytorch-lightning >=1.3.0
+    - requests
+    - scikit-learn
+    - statsmodels
+    - pytorch >=1.4
+    - torchinfo
+    - tqdm
+    - xlrd
+    # The following is necessary to pass pip-check
+    # pyDeprecate==0.3.1 is necessary for PL==1.5.10
+    - pyDeprecate 0.3.1
+    # pyppmd and zipfile-deflate constraints are necessary for py7zr
+    - pyppmd >=0.17.0,<0.18.0
+    - zipfile-deflate64 >=0.2.0
```

Comment:
Can you please add an issue in the `py7zr` feedstock to have this updated or make a PR there yourself?